content/post/articles/Music-and-silence.md

```diff
@@ -0,0 +1,57 @@
+
+The brain has to make efforts to analyze and process information based on abstract and logical reasoning during coding, several studies such as [University of Birmingham research](http://www.sciencedirect.com/science/article/pii/0003687072901019), [PNAS research](https://www.pnas.org/content/116/8/3310), [Stanford University School of Medicine research](https://www.sciencedirect.com/science/article/pii/S0896627307005004) show that music is effective in improving focus and productivity.
+
+### Concentration and productivity
+The benefits of music on our brain are numerous we can mention:
+###### Boost motivation
+
+I'm sure you know this good old mister "bug" that everyone wants to embrace he only brings us happiness when coding 🐛, more seriously as a developer, we often if not always encounter bugs in our code which can sometimes push us to abandon our code, because we cannot find the error, or because we consider that we have sought too much for the solution, in short it demotivates us but thanks to the studies cited above we can say that to listen a music that we like can help to take a step back, to reconsider the problem from another angle because the music will make us happy, more relaxed and therefore open to new possibilities to kill or permanently solve this dear mister bug 😊.
+
+###### Boost creativity
+
+Creativity can be boosted by music, according to a [study](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0182210) by researchers at Radboud University (Netherlands) and Sydney University of Technology (Australia) the researchers found that listening to happy music, which they define as music with positive, awakening values (a relative positivity because coders who love devil metal are also happy) facilitates more inspiring creative thinking than silence.
+
+I'm a supporter of this opinion, so here are some nuggets(without wanting to advertise) I use when I meet mister bug:
+- [RISE (ft. The Glitch Mob, Mako, and The Word Alive)](https://www.youtube.com/watch?v=fB8TyLTD7EE)
+- [Phoenix (avec Cailin Russo et Chrissy Costanza)](https://www.youtube.com/watch?v=i1IKnWDecwA)
+- [Take Over (avec Jeremy McKinnon (A Day To Remember), MAX, Henry)](https://www.youtube.com/watch?v=KbNL9ZyB49c)
+
+# Code with silence
+
+Lao Tzu said <<*Silence is a source of Great Strength*>>
+
+Not everyone uses music to cope with the different difficulties that one can encounter while coding so there are those who are for silence (the zone), a 2013 [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) found that two hours of silence could create new cells in the hippocampus region, a brain area linked to learning, remembering, and emotions.According to a 2006 [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.  
```

Comment:
```suggestion
Not everyone uses music to cope with the different difficulties that one can encounter while coding so there are those who are for silence (the zone), a 2013 [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) found that two hours of silence could create new cells in the hippocampus region, a brain area linked to learning, remembering and emotions. According to a 2006 [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.  
```

You should mention that this [study](https://www.researchgate.net/publication/259110014_Is_silence_golden_Effects_of_auditory_stimuli_and_their_absence_on_adult_hippocampal_neurogenesis) was conducted on mice, and the type of music involved is again, classical music. 

And this:
> [study](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1860846/), two minutes of silence relieves tension in the body and brain and is more relaxing than listening to music.  

Doesn't paint the complete picture, the articles specifically states that two minutes of silence *between* two music listening session is what causes relaxation. There are no mention of silence alone. So this is more an argument for both music and silence, with no clear cut.